launcher/src/main/go/go.mod

```diff
@@ -0,0 +1,24 @@
+module launcher
+
+go 1.23
+
+require (
+	github.com/elastic/go-sysinfo v1.15.0
+	github.com/fatih/color v1.18.0
+	github.com/kataras/jwt v0.1.12
+	github.com/spf13/afero v1.11.0
```

Comment:
This is filesystem abstraction that we use for testing. It doesn't bring google cloud SDK since we don't use this implementation. I've tidy go mod files and indeed these are not used.